src/modeler/AutoRest.Swagger/Validation/XMSPageableListByRGAndSubscriptions.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace AutoRest.Swagger.Validation
 {
-    public class XMSPageableListByRGAndSubscriptionsValidation : TypedRule<Dictionary<string, Schema>>
+    public class XMSPageableListByRGAndSubscriptions : TypedRule<Dictionary<string, Schema>>
```

Comment:
minor: If i remember correctly between these 50 commits, I've seen usage of XMS as `XMs***` & `Xms***`. If that is the case feel free to update rest for consistency.